web_search_autocomplete_prefetch/static/src/js/web_search_autocomplete_prefetch.js

```diff
@@ -83,6 +84,7 @@ openerp.web_search_autocomplete_prefetch = function(instance)
     });
 
     instance.web.search.AutoComplete.include({
+        keypress_timeout: 350,
```

Comment:
Hi. It could be interesting to have a setting for that value (ir.config;parameter) ? or not ? just a question.